src/goOutline.ts

```diff
@@ -156,10 +156,13 @@ function convertToCodeSymbols(
 			let start = byteOffsetToDocumentOffset(decl.start - 1);
 			let end = byteOffsetToDocumentOffset(decl.end - 1);
 			range = new vscode.Range(document.positionAt(start), document.positionAt(end));
+			if (decl.type === 'type') {
+				let line = document.lineAt(document.positionAt(start));
+				let regex = new RegExp('\\bstruct\\b');
```

Comment:
Can you make this regex stricter? Something like `RegExp('^\s*type\s+' + label + '\s+struct\s+ {')` ?

Also, update the test for the outline feature.

Reply:
Thanks. I guess you are referring to an older version of my code. But, as you pointed out even the newer exp is not very strict. Will fix it and fix the tests.

Reply:
For some reason, regex.test is always returning false if I use the above regex (IMO it is right). I even tried  ```let regex = new RegExp(`^\s*type\s${label}\s+struct\s*{`);```.  Still returns false.